hook/DeviceHook.js

```diff
@@ -616,7 +616,10 @@ class DeviceHook extends Hook {
     switch (type) {
       case "new_device":
         // no new device alarm on Firewalla
-        if (sysManager.isMyMac(host.mac)) return
+        if (sysManager.isMyMac(host.mac)) {
+          log.warn('New device alarm on Firewalla', host)
```

Comment:
no need to be log.warn, log.info is good enough.

log.warn will be displayed on production boxes by default